### DISCUSSION TITLE
Flowershow for Self-Publishing Alpha - Feedback Wanted 🙏

### DISCUSSION THREAD

**Discussion by PhilippeduPreez** [flowershow/flowershow#215] [Announcements]
We've just released the first alpha of Flowershow for self-publishing 🙌

Publish your markdown digital garden (obsidian-powered or otherwise) as fast as lighting ⚡

It's as simple as (once you have node installed):

```
npx flowershow@latest install
npx flowershow build-static

# then copy the build site folder to your favorite hosting service!
# cp flowershow-export-folder to-my-hosting-platform
```

You can read detailed instructions in our tutorial:

https://flowershow.app/docs/publish-tutorial

⛑️ Need assistance? Jump on our [discord chat channel][discord] and one of the team will be happy to help!

[discord]: https://discord.gg/vQ5Y2uUzt6  

## Feedback 📣

**Feedback wanted! 😄** we really want your comments and thoughts about what works, bugs and how we could improve!

You can share in the thread below or on our [discord chat channel][discord].

**Comment by rufuspollock** [comment]
When i did:

```
npx flowershow install
```

Got `command not found`

Need to do `npx flowershow@latest install` (i.e. add `@latest`) as i had old version of flowershow already installed.

This is probably unusual as most people won't have flowershow cli already installed 🙂

**Reply by olayway** [reply]
Added this to our self-publishing tutorial.

**Comment by davidleejy** [comment]
Would be nice if there were a simple live demo for users to experience.

Flowershow sounds like a great project!

**Reply by PhilippeduPreez** [reply]
@davidleejy Thanks for the feedback! That is definitely on the to-do list! Will link it here once available

**Reply by rufuspollock** [reply]
@davidleejy thanks for suggestion and feel free to add comments to #127 

Also what do you think of the "steps" (not a video i know) on the front page https://flowershow.app - is that useful?

**Reply by davidleejy** [reply]
> @davidleejy thanks for suggestion and feel free to add comments to #127
> 
> Also what do you think of the "steps" (not a video i know) on the front page https://flowershow.app - is that useful?

Those steps are useful! Excited and will be keeping an eye on Flower Show's development.

**Reply by rufuspollock** [reply]
@davidleejy we did a video! It's up on https://flowershow.app/docs/publish-tutorial

**Comment by rufuspollock** [comment]
Another UX bug is that currently we set up symlinks with *absolute* paths and these obviously break on deployment to a hosting service if you use e.g. git deploy with cloudflare, vercel, netlify etc.

**Reply by olayway** [reply]
Thanks for spotting that, I'll fix it asap!
